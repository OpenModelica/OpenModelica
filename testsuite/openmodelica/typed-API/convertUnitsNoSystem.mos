// status: correct

convertUnits("m", "mm"); getErrorString();
(_, r, _) := convertUnits("m", "mm"); getErrorString();
r;
s := String(r);
s;

// Result:
// (true,0.001,0.0)
// ""
//
// ""
// 0.001
// "0.001"
// "0.001"
// endResult
