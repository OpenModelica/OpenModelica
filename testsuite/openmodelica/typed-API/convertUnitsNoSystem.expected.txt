(true,0.001,0.0)
""

""
0.001
"0.001"
"0.001"
